pmm/pmm2-upgrade-tests.groovy

```diff
@@ -217,7 +217,7 @@ pipeline {
             }
             steps {
                 script {
-                    setupPMMClient(env.SERVER_IP, CLIENT_VERSION, 'pmm2', 'no', 'release', 'yes', 'compose_setup')
+                    setupPMMClient(env.SERVER_IP, CLIENT_VERSION, 'pmm2', 'no', 'release', 'yes', 'compose_setup', 'admin')
```

Comment:
Can we use variables here? We can add it to `environment` section:
```
        PMM_ADMIN_PASSOWRD=admin
```

Reply:
yes, good point, will modify thank you